.golangci.yaml

```diff
@@ -1,10 +1,84 @@
 run:
-  skip-dirs:
-      - pkg/vendorfakes
```

Comment:
I'm guessing the new rules make it okay to lint everything in the vendorfakes directory?

Reply:
Yes, it passed with this `skip-dirs` directive so I didn't bother to keep it there.